runtime/master/src/main/java/org/apache/nemo/runtime/master/RuntimeMaster.java

```diff
@@ -332,6 +331,28 @@ public void requestContainer(final String resourceSpecificationString) {
     }
   }
 
+  /**
+   * Requests a lambda executor.
+   */
+  public void requestLambdaExecutor() {
+    final ExecutorRepresenter executorRepresenter;
+    final String executorId = RuntimeIdManager.generateExecutorId();
+    final String nodeName = "192.168.0.100";
+
+    // LambdaRuntimeMaster manages Netty Server for LambdaExecutor to connect to
+    // Set up netty server at port 9999, and registers handler
+    this.lambdaMaster = new LambdaMaster();
```

Comment:
it would be good to use `port` parameter

Reply:
I think we have too many parameters in RuntimeMaster, making it harder to read and maintain.
In the future, we should allow LambdaMaster to search for available open port to use, so anyway we won't need a static port variable in RuntimeMaster.
So I think it's better to keep `port` only inside the LambdaMaster class.